packages/backend/src/auth/account/account.ts

```diff
@@ -174,3 +175,32 @@ export const fromSession = (
         },
     });
 };
+
+export const fromOauth = (
+    sessionUser: SessionUser,
+    token: {
+        accessToken: string;
+        accessTokenExpiresAt?: Date;
+        refreshToken?: string;
+        refreshTokenExpiresAt?: Date;
+        scope?: string[];
+        client: { id: string };
+    },
+): OauthAccount => {
+    const [organization, user] = extractOrganizationFromUser(sessionUser);
```

Comment:
can't this be extracted from the token?

Reply:
@stevelikesmusic can you review?

Reply:
We could, because the token contains the userUuid and orgUuid. in fact, we use those to find the session user in the middleware https://github.com/lightdash/lightdash/blob/main/packages/backend/src/controllers/authentication/middlewares.ts#L61 . The only downside is that we'll have to make an extra query to get org/user details to populate the account, I don't think there is any benefit on doing that.